pages/en/lb3/3.0-Release-Notes.md

```diff
@@ -30,6 +30,8 @@ dependency.
 
 See [loopback#275](https://github.com/strongloop/loopback/issues/275).
 
+Please see [how to migrate](http://loopback.io/doc/en/lb3/Migrating-to-3.0.html#loopback-datasource-juggler-is-a-regular-dependency-now)
```

Comment:
+1 for consistency.  JFYI in general, we omit "please" since it's superfluous in technical documentation....even for Canadians.  It's not a big deal, but in the future "please" omit it ;-)